content/ru/docs/tasks/administer-cluster/certificates.md

```diff
@@ -0,0 +1,280 @@
+
+   Обратите внимание, что команды необходимо адаптировать под архитектуру и используемую версию cfssl.
+
+   ```shell
+   curl -L https://github.com/cloudflare/cfssl/releases/download/v1.5.0/cfssl_1.5.0_linux_amd64 -o cfssl
+   chmod +x cfssl
+   curl -L https://github.com/cloudflare/cfssl/releases/download/v1.5.0/cfssljson_1.5.0_linux_amd64 -o cfssljson
+   chmod +x cfssljson
+   curl -L https://github.com/cloudflare/cfssl/releases/download/v1.5.0/cfssl-certinfo_1.5.0_linux_amd64 -o cfssl-certinfo
+   chmod +x cfssl-certinfo
+   ```
+
+1. Создайте директорию для хранения артефактов и инициализируйте cfssl:
+
+   ```shell
+   mkdir cert
+   cd cert
+   ../cfssl print-defaults config > config.json
+   ../cfssl print-defaults csr > csr.json
+   ```
+
+1. Создайте JSON-конфиг для генерации файла CA (например, `ca-config.json`):
+
+   ```json
+   {
+     "signing": {
+       "default": {
+         "expiry": "8760h"
+       },
+       "profiles": {
+         "kubernetes": {
+           "usages": [
+             "signing",
+             "key encipherment",
+             "server auth",
+             "client auth"
+           ],
+           "expiry": "8760h"
+         }
+       }
+     }
+   }
+   ```
+
+1. Создайте JSON-конфиг для запроса на подписание сертификата (CSR) (например,
+   `ca-csr.json`). Замените значения, помеченные угловыми скобками, на нужные.
+
+   ```json
+   {
+     "CN": "kubernetes",
+     "key": {
+       "algo": "rsa",
+       "size": 2048
+     },
+     "names":[{
+       "C": "<country>",
+       "ST": "<state>",
+       "L": "<city>",
+       "O": "<organization>",
+       "OU": "<organization unit>"
+     }]
+   }
+   ```
+
+1. Сгенерируйте ключ CA (`ca-key.pem`) и сертификат (`ca.pem`):
+
+   ```shell
+   ../cfssl gencert -initca ca-csr.json | ../cfssljson -bare ca
+   ```
+
+1. Создайте конфигурационный JSON-файл для генерации ключей и сертификатов для сервера API (например, `server-csr.json`). Замените значения, помеченные угловыми скобками, на нужные. `MASTER_CLUSTER_IP` – это cluster IP сервиса для сервера API, как описано в предыдущем подразделе. В приведенном ниже примере предполагается, что `cluster.local` используется в качестве доменного имени по умолчанию.
+
+   ```json
+   {
+     "CN": "kubernetes",
+     "hosts": [
+       "127.0.0.1",
+       "<MASTER_IP>",
+       "<MASTER_CLUSTER_IP>",
+       "kubernetes",
+       "kubernetes.default",
+       "kubernetes.default.svc",
+       "kubernetes.default.svc.cluster",
+       "kubernetes.default.svc.cluster.local"
+     ],
+     "key": {
+       "algo": "rsa",
+       "size": 2048
+     },
+     "names": [{
+       "C": "<country>",
+       "ST": "<state>",
+       "L": "<city>",
+       "O": "<organization>",
+       "OU": "<organization unit>"
+     }]
+   }
+   ```
+
+1. Сгенерируйте ключ и сертификат для сервера API (по умолчанию они сохраняются в файлах `server-key.pem` и `server.pem`, соответственно):
```

Comment:
```suggestion
1. Сгенерируйте ключ и сертификат для сервера API (по умолчанию они сохраняются в файлах `server-key.pem` и `server.pem` соответственно):
```

Reply:
I'm still not sure about the comma here.